packages/core/src/components/overlay/overlay.tsx

```diff
@@ -84,6 +84,11 @@ export interface IBackdropProps {
     backdropProps?: React.HTMLProps<HTMLDivElement>;
 
     /**
+     * Whether underlying content should be scrollable while the backdrop is visible.
+     */
+    backgroundScrollingDisabled?: boolean;
```

Comment:
i don't see a need for this prop. **an overlay with a backdrop should always prevent scrolling.** i can't see a case where you'd want a backdrop and the ability to scroll the document underneath.